src/crypto/tls/auth.go

```diff
@@ -123,10 +125,15 @@ func typeAndHashFromSignatureScheme(signatureAlgorithm SignatureScheme) (sigType
 		sigType = signatureECDSA
 	case Ed25519:
 		sigType = signatureEd25519
-	case EdDilithium3:
-		sigType = signatureEdDilithium3
 	default:
-		return 0, 0, fmt.Errorf("unsupported signature algorithm: %#04x", signatureAlgorithm)
+		scheme := circlSign.SchemeByTLSIdentifier(uint(signatureAlgorithm))
+		if scheme == nil {
+			return 0, 0, fmt.Errorf("unsupported signature algorithm: %#04x", signatureAlgorithm)
+		}
+		sigType = sigTypeByCirclScheme(scheme)
+		if sigType == 0 {
+			return 0, 0, errors.New("circl scheme not supported")
```

Comment:
Probably also use `fmt.Errorf` and list the scheme..

Reply:
Done